sys/net/link_layer/Kconfig.lorawan

```
@@ -99,17 +99,37 @@ config LORAMAC_DEV_EUI_DEFAULT
     help
         Configure the default device EUI for join procedure.
 
+config LORAMAC_APP_KEY_DEFAULT
+    string "Application key"
+    default "00000000000000000000000000000000"
+    help
+        Configure the default application key for join procedure.
+
+if GNRC_LORAWAN_1_1
```

As GNRC LoRaWAN is still not modelled in Kconfig, and we are moving the 1.1 to a pseudomodule, we can use here the `USEMODULE_`-prefixed symbol
```suggestion
if USEMODULE_GNRC_LORAWAN_1_1
```